src/autolabel/database/engine.py

```diff
@@ -1,15 +1,15 @@
 from typing import Any, Optional
+
 from sqlalchemy import create_engine
 from sqlalchemy.engine.base import Engine
-from os.path import join, expanduser
 
 DB_ENGINE = None
 
 # This creates one global ".autolabel.db" in your home directory.
 # Having one global SQLite database in ~ is a poor idea, since
 # SQLite cannot handle multiple simultaneous writes (if you have
 # several different labeling jobs going on).
-#DB_PATH = join(expanduser("~"), ".autolabel.db")
```

Comment:
this change is already merged. please rebase.

Reply:
this is different from that change: removed the unused import and added a space in the comment (which btw I think we should just remove the comment altogether?)

Reply:
okay fair.